acir/src/circuit/brillig.rs

```diff
@@ -22,8 +22,10 @@ pub enum BrilligOutputs {
 pub struct Brillig {
     pub inputs: Vec<BrilligInputs>,
     pub outputs: Vec<BrilligOutputs>,
-    /// Results of oracles/functions external to brillig like a database read
+    /// Results of oracles/functions external to brillig like a database read.
+    // Each element of this vector corresponds to a single foreign call but may contain several values.
```

Comment:
nit are we missing `/` here to make it a doc comment?

Reply:
This was intentional as I'm not sure it's something that needs to be known by a user but as I was piecing together how brillig foreign calls worked it wasn't immediately obvious. As it's only really needed by people working inside the ACVM (as we should abstract around this) then I think a normal comment is fine.

Reply:
Ok sounds good